docs/app-dev/indexing-transactions.md

```diff
@@ -109,3 +116,15 @@ a query to `/subscribe` RPC endpoint.
 
 Check out [API docs](https://docs.tendermint.com/master/rpc/#subscribe) for more information
 on query syntax and other options.
+
+## Querying Blocks Events
+
+You can query for a paginated set of blocks by their events by calling the
+`/block_search` RPC endpoint:
+
+```bash
+curl "localhost:26657/block_search?query=\"block.height > 10 AND valset.changed > 0\""
```

Comment:
This is a nicer example than above (showing real event types). Maybe you could also update `account.name` to something like `message.sender` or whatever this is

Reply:
List a complete list of what events are indexed by default, in addition to the ones from the sdk would be cool. Or maybe that belongs in the Cosmos SDK docs?

In either case, it should be easy to find, which is not the case now.

Reply:
Updated the event names to be more relevant and I also added a `Default Indexes` section.